lib/util/constants.dart

```diff
@@ -55,6 +58,10 @@ const CHECK_WEB3_CONTRACT_ADDRESS = [
   "KT1CPeE8YGVG16xkpoE9sviUYoEzS7hWfu39",
   "KT1U49F46ZRK2WChpVpkUvwwQme7Z595V3nt"
 ];
+
+const ANNOUNCEMENT_ID_WELCOME = "welcome";
```

Comment:
Is there any way to use `enum` instead of defining 3 constants?

Reply:
```enum AnnoucementID {
  WELCOME("welcome"),
  SUBSCRIBE("subscription"),
  PRO_CHAT("proChat");

  const AnnoucementID(this.value);
  final String value;
}```

I think using it will be more simple.

Reply:
there is, but every time we use it, we have to convert it to String. there're no switch case situation so i think it better to keep them as String